zerver/lib/export.py

```diff
@@ -626,6 +628,13 @@ def get_realm_config() -> Config:
         parent_key='realm_id__in',
     )
 
+    Config(
+        table='zerver_realmplayground',
+        model=RealmPlayground,
+        normal_parent=realm_config,
+        parent_key='realm_id__in',
+    )
```

Comment:
I can't remember -- do we need code on the `zerver/lib/import_realm.py` side as well?